nft/src/lib.rs

```diff
@@ -109,6 +116,28 @@ decl_storage! {
 		#[cfg(not(feature = "disable-tokens-by-owner"))]
 		pub TokensByOwner get(fn tokens_by_owner): double_map hasher(twox_64_concat) T::AccountId, hasher(twox_64_concat) (T::ClassId, T::TokenId) => Option<()>;
 	}
+	add_extra_genesis {
+		config(endowed_accounts): BTreeMap<GenesisClassData<T>, BTreeMap<T::AccountId, Vec<GenesisTokenData<T>>>>;
+
+		build(|config: &GenesisConfig<T>| {
+			config.endowed_accounts.iter().for_each(|(class_data, tokens)| {
+				let result = Module::<T>::create_class(&class_data.0, class_data.1.to_vec(), class_data.2.clone());
```

Comment:
```suggestion
				let class_id = Module::<T>::create_class(&class_data.0, class_data.1.to_vec(), class_data.2.clone()).expect("Create class cannot fail while building genesis");
```

Never ignore error silencely

Reply:
Thank you, I didn't think about `expect`